### DISCUSSION TITLE
notification option: ntfy.sh

### DISCUSSION THREAD

**Discussion by lephtHanded** [containrrr/watchtower#1691] [Feature Requests] [answered]
Hello and thanks!

I just started using [ntfy.sh](https://ntfy.sh) (selfhosted, 'natch) and was wondering if it's possible to use curl as a notification system.

options would be mostly the same as all the others...

**Comment by flowcool** [comment, marked as answer]
Hey,

Same for me and this is how I manage it with env variable:
        - WATCHTOWER_NOTIFICATIONS=shoutrrr
        - WATCHTOWER_NOTIFICATION_URL=generic+https://[YOUR DOMAIN]/[YOUR TOPIC]?title=WatchtowerUpdates
And I'm receiving notification each time ;)

**Reply by lephtHanded** [reply]
awesomesauce!  thank you, i'll give it a shot

**Reply by simonaronsson** [reply]
This looks like a solid solution. In case it turns out to not be enough, please raise an issue in the shoutrrr repo instead. Thanks!

**Reply by lephtHanded** [reply]
definitely - just had the opportunity to test is this past weekend... works awesome.
